### PR TITLE
save as .npz, with option to save as .mat

### PR DESCRIPTION
**Pull request by danyluik**

### CODE REVIEW

File: vertex/extract_metrics.py [hunk 1]
```diff
@@ -52,21 +58,36 @@ def load_vertex_data(df, column, n_subjects, n_vertex, mask=None):
...
+
+
+
 
 #LOAD CIVET MASK TO IDENTIFY MIDLINE/ CORPOS COLLOSUM REGION
 #IDENTIFY 'VALID VERTICES' - IE VERTICES NOT IN THIS REGION
 midline_mask = np.loadtxt(args.mask_file)
 valid_vertices = np.where(midline_mask==1)[0].tolist() # list of valid indices in civet .txt file
-invalid_vertices = np.where(midline_mask==0)[0].tolist() #list of invalid indices in civet .txt file
+#invalid_vertices = np.where(midline_mask==0)[0].tolist() #list of invalid indices in civet .txt file
```
Comment (gdevenyi) [inline vertex/extract_metrics.py:83]: What's going on here?

Comment (gdevenyi) [reply vertex/extract_metrics.py:83]: Why was it commented?

File: vertex/extract_metrics.py [hunk 2]
```diff
@@ -29,9 +30,14 @@
     '--mask_file', help='path to CIVET midline mask',required=True)
 group.add_argument(
     '--output_suffix', help='suffix to add to output file name before extension, e.g. _smoothed', type=str, default="")
+group.add_argument(
+    '--save_mat', help='option to save matrix as .mat, otherwise saves as .npz', required=False, action='store_true')
```
Comment (gdevenyi) [inline vertex/extract_metrics.py:34]: Default must remain the same as after adding this code, so it should default to save_mat.

Change option to --save_npz

**Review by gdevenyi — CHANGES_REQUESTED** [review]

File: vertex/extract_metrics.py (hunk 1 above)
Comment (danyluik) [reply vertex/extract_metrics.py]: Oh yeah, it isn't used anywhere in the script
Uncommented it for consistency

File: vertex/extract_metrics.py (hunk 2 above)
Comment (danyluik) [reply vertex/extract_metrics.py]: done

**Review by gdevenyi — CHANGES_REQUESTED** [review]
This looks fine.

Can you look up how to squash these two commits and force-push so we have one sensible change? (Mostly for your education :+1:)

**Review by gdevenyi — APPROVED** [review]

**Pull request merged**